README.md

```diff
@@ -82,7 +82,7 @@ Add the FH Android `aar` library in your `build.gradle` project
 
 ```
 dependencies {
-	compile 'com.feedhenry:fh-android-sdk:2.3.1'
+	compile 'com.feedhenry:fh-android-sdk:2.4.0'
```

Comment:
does this mean we need to update the readme file as well every time there is a new release? Is there a better way to do this?